pyaedt/icepak.py

```diff
@@ -412,8 +456,17 @@ def create_network_blocks(self, input_list, gravity_dir, top, assign_material=Tr
        
         Returns
         -------
-        type
-            Networks boundary objects.
+        list
+            List of boundary objects created.
+
+        Examples
+        --------
+
+        >>> box1 = icepak.modeler.primitives.create_box([1, 2, 3], [10, 10, 10], "NetworkBox2", "copper")
+        >>> box2 = icepak.modeler.primitives.create_box([4, 5, 6], [5, 5, 5], "NetworkBox3", "copper")
+        >>> blocks = icepak.create_network_blocks([["NetworkBox2", 20, 10, 3], ["NetworkBox3", 4, 10, 2]], icepak.GravityDirection.ZNeg, 1.05918, False)
```

Comment:
```suggestion
        >>> blocks = icepak.create_network_blocks([["NetworkBox2", 20, 10, 3], ["NetworkBox3", 4, 10, 2]],
                                                   icepak.GravityDirection.ZNeg, 1.05918, False)
```
[PEP8](https://www.python.org/dev/peps/pep-0008/#id19) standard line length is 79, unless it's a docstring, where it's 72.  I'm ok with exceeding both of these by a bit, but 152 is a little extreme.

Reply:
No problem, thanks for catching it.